java-snapshot-testing-core/src/main/java/au/com/origin/snapshots/Snapshot.java

```diff
@@ -25,13 +27,14 @@
     private final Class<?> testClass;
     private final Method testMethod;
     private final Object[] current;
+    private final boolean isCi;
```

Comment:
Can this be `isCI` - just cause CI is an acronym like HTML.

Reply:
Makes sense. Will rename.